src/Sylius/Component/Order/Model/OrderInterface.php

```diff
@@ -114,6 +114,7 @@ public function getState(): string;
 
     /**
      * @param string $state
+     * @deprecated this method is deprecated since Sylius 1.3 and will be removed in Sylius 2.0, use state machine to manage order state
```

Comment:
We need to trigger deprecation errors in our implementations of these methods.

Reply:
Done 🎉